### ISSUE TITLE
rpmbuild fails for 6.1.0

### ISSUE THREAD

**Issue by dirtyharrycallahan** [adaptivecomputing/torque#406]
I was able to build 6.0 RPMs from the tar ball but it seems a couple of new files are unpackaged.

```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/pcallahan/rpmbuild/BUILDROOT/torque-6.1.0-1.adaptive.el7.x86_64
error: Installed (but unpackaged) file(s) found:
   /etc/ld.so.conf.d/torque.conf
   /etc/profile.d/torque.csh
   /etc/profile.d/torque.sh


RPM build errors:
    Installed (but unpackaged) file(s) found:
   /etc/ld.so.conf.d/torque.conf
   /etc/profile.d/torque.csh
   /etc/profile.d/torque.sh
[pcallahan@localhost Downloads]$

**Comment by backelj** [comment]
Hi, just wondering whether you've managed to resolve this issue?

**Comment by jpuskar** [comment]
Same here:
`Checking for unpackaged file(s): /usr/lib/rpm/check-files /root/rpmbuild/BUILDROOT/torque-6.1.0-1.adaptive.el7.centos.x86_64
error: Installed (but unpackaged) file(s) found:
   /etc/ld.so.conf.d/torque.conf
   /etc/profile.d/torque.csh
   /etc/profile.d/torque.sh


RPM build errors:
    Installed (but unpackaged) file(s) found:
   /etc/ld.so.conf.d/torque.conf
   /etc/profile.d/torque.csh
   /etc/profile.d/torque.sh`

I'm building from bento/centos7.3 with the following:

`sudo su

yum install -y epel-release
yum install -y python-pip
yum install -y python-devel
yum install -y python-crypto
yum install -y libffi-devel
yum group install -y "Development Tools"
yum install -y openssl-devel

cd /vagrant
pip install --upgrade pip
pip install pyconfig
pip install ansible
cd /root
ansible-playbook /vagrant/ansible_torque/ansible_torque/rpmbuild_torque.yml`

Here's the playbook:
`---
- hosts: localhost
  tasks:
    - name: download torque source
      get_url:
        url: http://wpfilebase.s3.amazonaws.com/torque/torque-6.1.0.tar.gz
        dest: /root/torque-6.1.0.tar.gz

    - name: install packages for building torque
      package:
        name: "{{ item }}"
        state: installed
      with_items:
        - 'libxml2-devel'
        - 'boost-devel'`

Then calling `cd /root; rpmbuild -tb ./torque-6.1.0.tar.gz`

**Comment by acvizi** [comment]
This issue should be resolved in upcoming releases of 6.0.4 and 6.1.2:

https://github.com/adaptivecomputing/torque/commit/091f7d14b01045ccb082c374ad1e4d1d8787913e
https://github.com/adaptivecomputing/torque/commit/49758a6fcad2285a13427e4fe1fd2133a8322a67

Chad Vizino
Adaptive Computing
